buildbot.mariadb.org/scripts/deb-install.sh

```diff
@@ -1,18 +1,29 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2154
 
-set -ex
+set -e
+
+# load common functions
+# shellcheck disable=SC1091
+. ./bash_lib.sh
+
+# function to be able to run the script manually (see bash_lib.sh)
+manual_run_switch "$1"
+
+set -x
+
+# print disk usage
 df -kT
 
-# show installed packages (//TEMP see below)
+# show installed packages
 dpkg -l | grep -iE 'maria|mysql|galera' || true
 
 # we want a clean installation here (//TEMP should already be the case, do better here!)
 sudo apt-get purge -y "$(dpkg -l | grep -iE 'maria|mysql|galera' | awk '{print $2}')"
 
 # check for needed commands
 for cmd in wget gunzip; do
-  command -v $cmd >/dev/null || { echo "need $cmd" && exit 1; }
+  command -v $cmd >/dev/null || { bb_log_err "$cmd command not found" && exit 1; }
 done
 
 # setup repository
```

Comment:
`$branch` below needs to be `$master_branch` otherwise it will be looking for `bb-10.2-release` as as the branch name. Same for `rpm-install.sh`.

Reply:
good catch!